api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/convolutional/AbstractConv.kt

```diff
@@ -19,6 +19,35 @@ import org.tensorflow.op.core.Variable
 import java.lang.IllegalArgumentException
 import kotlin.math.roundToInt
 
+/**
+ * Abstract Convolutional layer is a base block for building base types of convolutional layers
+ * of any dimensionality. It should simplify the internal calculations needed in most of
+ * the convolutional layers and abstract the process of naming weights for these layers. It keeps
+ * the actual implementation of convolutional layers i.e. the kernel and bias learnable variables
+ * that should be used in child classes in actual implementations of these layers. If the child class
+ * uses some values for its implementation in other form than it is kept in this child class,
+ * then this abstract class `internal` properties should keep the implementation values
+ * while the child class properties should keep the printable values that are more representative.
+ * But in most cases the `internal` and child values will be the same.
+ *
+ * @property filtersInternal number used by default in calculation of layer weights and i/o shapes
+ * @property kernelSizeInternal numbers used by default in calculation of layer weights and i/o shapes
+ * @property stridesInternal numbers used by default in calculation of layer weights and i/o shapes
+ * @property dilationsInternal numbers to keep for the dilations for implementation
+ * @property activationInternal activation used in [forward] operation implementation
+ * @property kernelInitializerInternal kernelInitializer used in actual kernel variable filling implementation
+ * @property biasInitializerInternal biasInitializer used in actual bias variable filling implementation
```

Comment:
strange to have twice kernelInitializerInternal kernelInitializer 
Better to have something like that "[kernelInitializer] An initializer for the convolution kernel."

Reply:
done